start-console-tls.sh

```diff
@@ -45,13 +45,13 @@ echo "Console Platform: $CONSOLE_IMAGE_PLATFORM"
 if [ -x "$(command -v podman)" ]; then
     if [ "$(uname -s)" = "Linux" ]; then
         # Use host networking on Linux since host.containers.internal is unreachable in some environments.
-        BRIDGE_PLUGINS="${PLUGIN_NAME}=https://localhost:9443"
+        BRIDGE_PLUGINS="${PLUGIN_NAME}=https://localhost:9444"
```

Comment:
Why did you change the localhost port from `9443 to 9444`? 
Could you please explain the reason for this change?

Reply:
Sure, the reason is that the `api-server` starts on 9443 on another process now, so this port is already taken on the machine. The bridge has to use another one. I've selected 9444 for that purpose. The value itself doesn't really matter, since it's only used for the bridge (it's not going to change the url your using to access the tls console).